CondCore/JetMETPlugins/test/testJetResolutionPayloadInspector.cpp

```diff
@@ -52,7 +52,7 @@ int main(int argc, char** argv) {
   histo4.process(connectionString, PI::mk_input(tag_eta, start, end));
   std::cout << histo4.data() << std::endl;
 
-  std::cout << "## Jet SF Histograms" << std::endl;
+  std::cout << "## Jet SF vs. Eta Histograms" << std::endl;
```

Comment:
Can we void using the `cout`s?

Reply:
This is simply a copy from the example unit test code provided. Surely I can replace them with LogINFO.